public/js/main.js

```diff
@@ -157,8 +157,37 @@ btnToggleAll.addEventListener("click", () => {
     });
   }
   saveData(todoList);
-  render();
+  renderData();
+});
+
+function renderData() {
+  const selectedFilter = document.getElementsByClassName('selected')[0];
+  let redirect = selectedFilter.getAttribute("href");
```

Comment:
`redirect` won't be reassigned, so let's use `const` here

- [ ] use `const`

- [ ] refactor variable name: `redirect` -> `selectedFilterRedirector`